vite.config.js

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
```

Comment:
I can be wrong, but if you use inline config this file become useless

I'm not the maintainer of these repo, but I think `vite.config.js` (using config with condition) is more preferable variant than inline configs